database/sqlite/create-schema.migrations.sql

```diff
@@ -0,0 +1,42 @@
+CREATE TABLE drs_object (
+    id INTEGER PRIMARY KEY,
```

Comment:
an int for `id` won't be sufficient here, since these ids are internet-facing and the group has expressed interest in aiming for global uniqueness. Let's make these UUIDs. In sqlite I think we have to represent this as `TEXT`, but please check that there isn't a UUID data type. foreign key statements will also need to be updated with the new datatype